### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by BohdanBBG**

### CODE REVIEW

**Review by SerjKocherga — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+ -create an email account for the notification service and write it to .env file
+ -use our scripts to create and fill the database with test data
+ 
+### How to run local
+
+- open CharlieBackend.sln
+- run your local or remote data base
+- run with IDE or use cmd "**dotnet run**" command in every folder
+  where we have **.csproj** file (for now it is AdminPanel and API)
+- open _https://localhost:5001_ to view API Swagger in a browser
```
Comment (SnipeFrost) [inline README.md:46]: isnt our api have port 5000?

**Review by SnipeFrost — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+* netcoreapp 3.1
+* MySql 8.0.21
+
+### Setup
+
+- clone this repo to your local machine using git clone
+- create an .env file to manage your application secrets:
+  https://github.com/tonerdo/dotnet-env
+ -create an email account for the notification service and write it to .env file
+ -use our scripts to create and fill the database with test data
```
Comment (SnipeFrost) [inline README.md:38]: Maybe need to say 
"Add connection string for created database to .env file"

File: README.md (hunk 1 above)
Comment (BohdanBBG) [reply README.md:46]: Yeah, you are right, thanks

File: README.md (hunk 2 above)
Comment (BohdanBBG) [reply README.md:38]: Right, thanks

**Review by SnipeFrost — APPROVED** [review]

**Review by whereismyroot — APPROVED** [review]

**Review by x1mx1x — APPROVED** [review]

**Review by sober607 — APPROVED** [review]

**Review by Vlad1023 — APPROVED** [review]

**Pull request merged**